scripts/models.py

```diff
@@ -4,13 +4,36 @@
 # we can parse out the JSON using the old code and filter on Workflow objects
 import os
 import sys
+from dataclasses import dataclass
 from pathlib import Path
+from typing import List, Optional
 
 import requests
 
 model_types = {"workflows", "events"}
 
 
+@dataclass(frozen=True, order=True)
+class ObjectReference:
```

Comment:
Minor but can we call it an `ImportReference` or something?

Reply:
For sure!